junit5/src/main/kotlin/io/dbkover/junit5/DBKoverExtension.kt

```diff
@@ -23,7 +23,9 @@ class DBKoverExtension : BeforeAllCallback, BeforeTestExecutionCallback, AfterTe
         }
 
         val dbKoverDataSet = context.findAnnotationThrowing(DBKoverDataSet::class.java)
-        context.getExecutor().beforeTest(dbKoverDataSet.path)
+
+        val paths = dbKoverDataSet.path.takeIf { it.isNotBlank() }?.let { listOf(it) } ?: dbKoverDataSet.paths.toList()
```

Comment:
Should we add some validation as `path` and `paths are mutual exclusive? 🤔

Reply:
Yes, I think it is a good idea to throw an exception if using both fields at the same time.

Reply:
Done 👍